tools/__tasks__/compile/javascript/babel.js

```diff
@@ -7,7 +7,13 @@ module.exports = {
     task: () =>
         execa(
             'babel',
-            [`${src}/javascripts`, '--out-dir', `${transpiled}/javascripts`],
+            [
+                `${src}/javascripts`,
+                '--out-dir',
+                `${transpiled}/javascripts`,
+                '--ignore',
+                '**/*.spec.js',
```

Comment:
it's working for me – i've added a fake test to make sure it's not a weird env thing, want to pull latest and try? will see how TC goes too